src/Symfony/Bridge/Doctrine/Tests/Validator/Constraints/UniqueEntityValidatorTest.php

```diff
@@ -664,23 +664,6 @@ public function testValidateInheritanceUniqueness()
             ->assertRaised();
     }
 
-    /**
-     * @expectedException \Symfony\Component\Validator\Exception\ConstraintDefinitionException
-     * @expectedExceptionMessage The "Symfony\Bridge\Doctrine\Tests\Fixtures\SingleStringIdEntity" entity repository does not support the "Symfony\Bridge\Doctrine\Tests\Fixtures\Person" entity. The entity should be an instance of or extend "Symfony\Bridge\Doctrine\Tests\Fixtures\SingleStringIdEntity".
-     */
-    public function testInvalidateRepositoryForInheritance()
```

Comment:
see my other comment: I think there was a reason for this: https://github.com/symfony/symfony/pull/15002

Reply:
we should add new tests for sure.